backend/src/main/java/com/bakdata/conquery/models/query/queryplan/ConceptQueryPlan.java

```diff
@@ -197,4 +189,62 @@ public boolean isOfInterest(Bucket bucket) {
 	public Set<TableId> collectRequiredTables() {
 		return child.collectRequiredTables();
 	}
+
+	public enum DateAggregationMode {
```

Comment:
Doku bitte, und schieb beide am besten in ein eigenes Paket und eigenen Dateien, die sind sehr essentiell und haben viele Folgen.